spec/videos/video_spec.rb

```diff
@@ -19,6 +19,7 @@
       it { expect(video.length).to be_a(Float) }
       it { expect(video.picture).to be_a(String)}
       it { expect(video.page_name).to be_a(String) }
+      it { expect(video.page_id).to be_a(String) }
```

Comment:
@claudiofullscreen - this method is analagous to page_name, though - if you use the Funky::Video find/where methods of the API, you will get `Funky::ContentNotFound` or an `[]`